infrastructure/main.tf

```diff
@@ -109,19 +115,56 @@ data "azurerm_key_vault" "s2s_vault" {
 
 data "azurerm_key_vault_secret" "s2s_key" {
   name      = "microservicekey-em-stitching-api"
-  key_vault_id = "${data.azurerm_key_vault.s2s_vault.id}"
+  vault_uri = "https://s2s-${local.local_env}.vault.azure.net/"
```

Comment:
https://cloudblogs.microsoft.com/opensource/2019/02/12/whats-new-in-the-latest-release-terraform-azure-provider-v1-22-0/
We should be using `key_vault_id` as vault_uri is deprecated